features/host_can_fill_listing_form.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Host can fill in a listing form 
+  As a Host,
+  In order to display information about myself,
+  I need to be able to fill in a listing form.
+
+  Scenario: Host can succesully view and fill in all fields in a listing form, without creating it
+    Given I visit the "Become a host" page
+    Then I should see "Create new listing"
+    And I fill in "name" with "George"
+    And I fill in "location" with "Stockholm"
+    And I fill in "description" with "Hello, please lend me your cat"
+    And I fill in "availability" with "All summer"
+    And I fill in "email" with "george@craft.se"
+    And I fill in "phone number" with "07071234567"
+    And I should see "Create Listing" button
+    Then I should not see "listing was created succesfully"
```

Comment:
`not`???

Reply:
Since this PR is about creating the form and making sure fields are visible, and not creating an actual listing, we wanted a then block to close the scenario with a message we can use for CREATING a listing in the future. But realize we could maybe re-think this step to make it more "confirming" of THIS specific scenario.. :)

Reply:
i think we can just remove that step. Because later we will implement the flash messages and then the test will fail. Unnecessary, imo.

Reply:
I agree. I also think that it's a bit too granular to make 2 PR's from a create feature. But it's your call.